x509-cert/src/builder.rs

```diff
@@ -92,6 +92,14 @@ pub enum Profile {
         enable_key_agreement: bool,
         /// should the key encipherment flag of KeyUsage be enabled
         enable_key_encipherment: bool,
+        #[cfg(feature = "hazmat")]
```

Comment:
I think it's a bit more idiomatic to put attributes after doc comments (or at least, that's what `std` does)

Reply:
Swapped that. Thank you!